### DISCUSSION TITLE
The issue is probably me

### DISCUSSION THREAD

**Discussion by me-chiel** [forntoh/LcdMenu#69] [General]
I'm working with your great library together with an encoder. 
Things are moving, and happening.

But if I hover over a menu item, and push the encoder button all I can think of is to use menu.enter(); to use that menu choice.
but how can I go to any subroutine from there?
is it only possible with an ItemToggle?

since I use an encoder I added a submenu line "Back" 
But after several tries I can't seem to get back to the main menu (menu.back(); wasn't accepted sadly) :(

sorry for this obvious questions, but if someone could help, that would be super nice.

**Comment by forntoh** [comment]
Hi, you can use ItemCommand
see https://forntoh.gitbook.io/lcdmenu/reference/menuitem#itemcommand
